### PR TITLE
fix: Patch of `avm/res/relay/namespace` to enforce the publishing

### PR DESCRIPTION
**Pull request by krbar**
## Description

Patch of `avm/res/relay/namespace` to enforce the publishing

## Pipeline Reference

| Pipeline |
| -------- |
|  [avm.res.relay.namespace](https://github.com/krbar/bicep-registry-modules/actions/workflows/avm.res.relay.namespace.yml) |

## Type of Change

- [ ] Update to CI Environment or utlities (Non-module effecting changes)
- [x] Azure Verified Module updates:
  - [x] Bugfix containing backwards compatible bug fixes, and I have NOT bumped the MAJOR or MINOR version in `version.json`:
    - [ ] Someone has opened a bug report issue, and I have included "Closes #{bug_report_issue_number}" in the PR description.
    - [ ] The bug was found by the module author, and no one has opened an issue to report it yet.
  - [ ] Feature update backwards compatible feature updates, and I have bumped the MINOR version in `version.json`.
  - [ ] Breaking changes and I have bumped the MAJOR version in `version.json`.
  - [ ] Update to documentation

## Checklist

- [x] I'm sure there are no other open Pull Requests for the same update/change
- [x] I have run `Set-AVMModule` locally to generate the supporting module files.
- [x] My corresponding pipelines / checks run clean and green without any errors or warnings

### CODE REVIEW

File: avm/res/relay/namespace/main.json [hunk 1]
```diff
@@ -447,7 +447,7 @@
         }
       ],
       "metadata": {
-        "description": "Optional. Authorization Rules for the Relay namespace."
+        "description": "Optional. Authorization Rules for the Relay Namespace."
```
Comment (eriqua) [inline avm/res/relay/namespace/main.json:450]: Shouldn't this update the templateHash?

Comment (krbar) [reply avm/res/relay/namespace/main.json:450]: Interesting, but it didn't. And the pipeline finishes green...

Comment (krbar) [reply avm/res/relay/namespace/main.json:450]: I just re-ran the Set-AVMModule, no changes....

**Review by AlexanderSehr — APPROVED** [review]

File: avm/res/relay/namespace/main.json (hunk 1 above)
Comment (AlexanderSehr) [reply avm/res/relay/namespace/main.json:450]: @eriqua it doesn't because it's just a casing change. Just tested via `bicep build` myself.

**Pull request merged**